src/test/munit/xa/activemq-xa-config.xml

```diff
@@ -23,4 +23,13 @@ http://www.mulesoft.org/schema/mule/jms http://www.mulesoft.org/schema/mule/jms/
         </jms:active-mq-connection>
     </jms:config>
 
+    <jms:config name="JMS_Config_XA_MAX_DOCKER_BROKER">
```

Comment:
Avoid calling "Docker" you can provide an instance of the broker not using Docker.